config/initializers/friendly_id.rb

```diff
@@ -0,0 +1,88 @@
+# FriendlyId Global Configuration
```

Comment:
I think maybe leave it - it's got some non-commented cruft that 's maybe required or at least sensible for the latest version of the gem ...